bin/hwinj/pycbc_generate_hwinj

```diff
@@ -294,19 +304,27 @@ sngl.spin2y = opts.spin2y
 sngl.spin2x = opts.spin2x
 
 # generate waveform
-logging.info('Generating waveform at %.3fMpc beginning at %.3fHz for SNR calculation', sim.distance, opts.psd_low_frequency_cutoff)
-h_plus, h_cross = get_td_waveform(sim, approximant=name, phase_order=phase_order, f_lower=opts.psd_low_frequency_cutoff,
-                      delta_t=1.0/sample_rate)
+logging.info('Generating waveform at %.3fMpc beginning at %.3fHz for '
+             'SNR calculation', sim.distance, opts.psd_low_frequency_cutoff)
+h_plus, h_cross = get_td_waveform(sim, approximant=name,
+                                  phase_order=phase_order,
+                                  f_lower=opts.psd_low_frequency_cutoff,
```

Comment:
Because it matches the low-frequency cutoff for the SNR calculation. Later at the end, it writes the waveform to file and that uses ``--waveform-low-frequency-cutoff``. Was this way because in ER8/9 we only searched from 30Hz up so I wanted to calculate the SNR from 30Hz up but generate the waveform lower at 15Hz.